php/Plugin.php

```diff
@@ -164,7 +186,7 @@ public function asset_version() {
 	public function meta( $field = null ) {
 		static $meta;
 
-		if ( ! isset( $meta ) ) {
+		if ( ! isset( $meta ) && function_exists( 'get_plugin_data' ) ) {
```

Comment:
Could we switch to using `get_file_data()` which appears to be [always available](https://github.com/WordPress/WordPress/blob/20e781f44d0e062be0f541cb75011f3c6c6d2006/wp-includes/functions.php#L5690-L5730)?

Reply:
We could but `get_plugin_data` does a better job at sane defaults.

If we continue to use it we need to add:
```
if ( ! function_exists( 'get_plugin_data' ) ) {
    require_once( ABSPATH . 'wp-admin/includes/plugin.php' );
}
```